### ISSUE TITLE
Add boundary=protected_area

### ISSUE THREAD

**Issue by pastk** [organicmaps/organicmaps#2926]
OM has support for `boundary=national_park` and `leisure=nature_reserve`, but not for [`boundary=protected_area`](https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dprotected_area) which has about the same number of uses (90+k).
(and maybe `boundary=aboriginal_lands` too)

Please add support for associated keys [`protect_class=`](https://wiki.openstreetmap.org/wiki/Key:protect_class) and [`protection_title=`](https://wiki.openstreetmap.org/wiki/Key:protection_title) at the same time.

**Comment by vng** [comment]
We have a claim from a user that leisure-nature_reserve (boundary-protected_area-2) is almost not visible on the map.
We draw light green hatching.

**Comment by pastk** [comment]
I think on lower zooms the hatching should be more opaque / visible and disappear completely on detailed zooms when we're inside of the area already (because the hatching makes it harder to read the map).

**Comment by pastk** [comment]
> I think on lower zooms the hatching should be more opaque / visible and disappear completely on detailed zooms when we're inside of the area already (because the hatching makes it harder to read the map).

The biggest challenge with this approach is that very big reserves are not uncommon and they won't fit on the screen even on the lowest zoom (z10). In these cases making the hatching pattern more prominent will severely reduce map readability on overview zooms, e.g. for overviewing hiking routes going through the park.
(e.g. "Adirondack Park" in the northern part of New York state, USA)

So a better solution might be to use a distinct border line, which should be asymmetric to hint which side is "inner, a reserve" and which is "outer, no reserve restrictions".

A sample from Mapy.cz (not distinct enough for my taste, but gives an idea)
![image](https://github.com/organicmaps/organicmaps/assets/18434508/da5d2592-6c40-4b46-a34b-e5229f066f9e)

**Comment by deevroman** [comment]
Has support for this tag been added? I think it's because of this, and because of this, in some cities, cultural heritage areas have begun to be displayed as nature reserves

https://www.openstreetmap.org/#map=14/51.6704/39.2086

<table>
<tr>
 <td>

![изображение](https://github.com/organicmaps/organicmaps/assets/25708359/0f165ab9-aec9-496e-b45b-2a60685a6ac0)


<td>

<img src="https://github.com/organicmaps/organicmaps/assets/25708359/2b655adf-0996-4cf4-8c67-b3d07c0963b0">


</td> 

</tr>

</table>

 <td>

<img width="1067" alt="Снимок экрана 2024-06-19 в 14 30 59" src="https://github.com/organicmaps/organicmaps/assets/25708359/f5653acd-aa37-401d-b67f-46bb9625578f">

https://www.openstreetmap.org/relation/4136162

**Comment by pastk** [comment]
Yep they were added awhile ago, but only protect_class=1/1a/1b has a green hatching fill like reserves https://github.com/organicmaps/organicmaps/pull/5368#issuecomment-1598695209

**Comment by deevroman** [comment]
But why is the tree icon visible in this case?

**Comment by pastk** [comment]
> But why is the tree icon visible in this case?

Thats the way it was done. I guess because we use this icon for all nature reserves.



> p. s. If you open [OSM notes](https://www.openstreetmap.org/note/4297731) in the specified location, you will see that the current behavior has confused the user

Yeap I see. It is a protected zone https://www.openstreetmap.org/relation/4128162#map=17/51.67410/39.21022
`The tag [protect_class](https://wiki.openstreetmap.org/wiki/Key:protect_class)=22 specifies areas which concerns cultural and historical heritage, especially as an ensemble of objects and/or areas, which includes protection under mixed category and criteria of natural, architectural, historical and cultural value.`

Perhaps we should use different visualization and translations for different protect_class.
(or maybe drop some of them completely if showing them doesn't make sense)
That's why I've change issues' title already.

It'd be helpful if someone could research misc protect classes and suggest how we should treat them.

**Comment by ntzm** [comment]
I don't think protect_class=22 should be rendered as a tree:

> Cultural assets (cultural assets and - efforts, districts historical heritage, monument conservation):
protection of sites with special architecture or historic interest, designed and created intentionally by people: urban layout, buildings, landscape, gardens, art-goods, ...

As it appears in urban areas

**Comment by kesapaiva** [comment]
> I don't think protect_class=22 should be rendered as a tree:
> 
> > Cultural assets (cultural assets and - efforts, districts historical heritage, monument conservation):
> > protection of sites with special architecture or historic interest, designed and created intentionally by people: urban layout, buildings, landscape, gardens, art-goods, ...
> 
> As it appears in urban areas

And here's why:

<img width="550" src="https://github.com/user-attachments/assets/23dc1720-b9f7-4cb4-95d2-18577f3fc45f"/>
